From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igor.pecovnik@gmail.com>
Date: Tue, 27 Feb 2018 16:21:49 +0100
Subject: [ARCHEOLOGY] RK3328 kernel and u-boot upstream patches and few small
 fixes

> X-Git-Archeology: - Revision 9e32020d1aa974462d82d5c404e305da04bdb432: https://github.com/armbian/build/commit/9e32020d1aa974462d82d5c404e305da04bdb432
> X-Git-Archeology:   Date: Tue, 27 Feb 2018 16:21:49 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: RK3328 kernel and u-boot upstream patches and few small fixes
> X-Git-Archeology:
> X-Git-Archeology: - Revision 7d2f3af08f23049c91c88eec5062613bbfbc85d4: https://github.com/armbian/build/commit/7d2f3af08f23049c91c88eec5062613bbfbc85d4
> X-Git-Archeology:   Date: Thu, 24 May 2018 15:44:15 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Merging Rockchip family
> X-Git-Archeology:
> X-Git-Archeology: - Revision 41b589372f2c1a8450033db78f3d8ed9db1b2196: https://github.com/armbian/build/commit/41b589372f2c1a8450033db78f3d8ed9db1b2196
> X-Git-Archeology:   Date: Sun, 26 Aug 2018 14:14:14 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Creating new family rockchip64. 3288 and 3399 was merged in, except Nanopi which stays in RK3399 family with sources attached to FA repository until we manage to join that in. Enabled Nanopc T4, Nanopi M4, Renegade. Tested on Rock64 and Nanopc T4.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 42ecf1582f83241e18ce329193c5eac66fd244de: https://github.com/armbian/build/commit/42ecf1582f83241e18ce329193c5eac66fd244de
> X-Git-Archeology:   Date: Fri, 03 Jan 2020 11:37:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Move rockchip64 legacy to most recent kernel. Tested with standard https://github.com/armbian/autotests stress test on RockPro64 and Rock64.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5597ecfe073a70e11a3d832ca210471217706a0f: https://github.com/armbian/build/commit/5597ecfe073a70e11a3d832ca210471217706a0f
> X-Git-Archeology:   Date: Fri, 03 Jan 2020 11:42:56 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move rockchip64 legacy to most recent kernel. Tested with standard https://github.com/armbian/autotests stress test on RockPro64 and Rock64. (#1714)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 scripts/package/builddeb | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index de4bfb55e128..530f310b4c5b 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -293,7 +293,7 @@ if [ "$ARCH" = "um" ]; then
 	cat <<EOF >> debian/control
 
 Package: $packagename
-Provides: linux-image, linux-image-rockchip, linux-image-rockchip-4.4, linux-modules-$version
+Provides: linux-image, linux-image-2.6, linux-modules-$version
 Architecture: any
 Description: User Mode Linux kernel, version $version
  User-mode Linux is a port of the Linux kernel to its own system call
@@ -310,7 +310,7 @@ else
 	cat <<EOF >> debian/control
 
 Package: $packagename
-Provides: linux-image, linux-image-rockchip, linux-image-rockchip-4.4, linux-modules-$version
+Provides: linux-image, linux-image-2.6, linux-modules-$version
 Suggests: $fwpackagename
 Architecture: any
 Description: Linux kernel, version $version
@@ -366,7 +366,7 @@ cat <<EOF >> debian/control
 
 Package: $libc_headers_packagename
 Section: devel
-Provides: linux-kernel-headers, linux-kernel-headers-rockchip, linux-kernel-headers-rockchip-4.4
+Provides: linux-kernel-headers
 Architecture: any
 Description: Linux support headers for userspace development
  This package provides userspaces headers from the Linux kernel.  These headers
@@ -396,7 +396,7 @@ if [ -n "$BUILD_DEBUG" ] ; then
 
 Package: $dbg_packagename
 Section: debug
-Provides: linux-debug, linux-debug-$version, linux-debug-rockchip, linux-debug-rockchip-4.4
+Provides: linux-debug, linux-debug-$version
 Architecture: any
 Description: Linux kernel debugging symbols for $version
  This package will come in handy if you need to debug the kernel. It provides
-- 
Armbian

